db/migrate/20230119040224_add_users_ref_to_comment.rb

```diff
@@ -0,0 +1,5 @@
+class AddUsersRefToComment < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
```

Comment:
- [ ] Taking into consideration the given ERD, we can see that `user` model is referenced using foreign key in models as `AuthorId`. Let's, therefore, bear in mind that `author` is aliasing the `user` model, and hence, our migration query should be similar to:

```ruby
...
  def change
    add_reference :comments, :author, references: :users, null: false, foreign_key: {to_table: :users}
  end
...

```

or.. 

```ruby
  create_table :comments do |t|
    . . .
    t.references :author, foreign_key: { to_table: 'users' } 
    . . .
  end
```

Reply:
well resolved! ✔️